app/src/main/AndroidManifest.xml

```diff
@@ -9,10 +9,11 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.MensFashion">
+        android:theme="@style/RegistrationTheme">
         <activity
             android:name=".ui.MainActivity"
-            android:exported="false">
+            android:exported="true"
+            android:screenOrientation="portrait">
```

Comment:
I didn't make the landscape design for registration be I didn't know if it's required or not, so if it's required please tell me to do it.